chatkit-core/src/main/kotlin/com/pusher/chatkit/state/ChatState.kt

```diff
@@ -1,18 +1,25 @@
 package com.pusher.chatkit.state
 
 import com.pusher.chatkit.rooms.state.JoinedRoomsState
+import com.pusher.chatkit.users.state.UserInternalType
 
 internal data class ChatState(
-    val joinedRoomsState: JoinedRoomsState?
+    val joinedRoomsState: JoinedRoomsState?,
+    val currentUser: UserInternalType?
```

Comment:
currentUser should be first, per the [specs](https://github.com/pusher/chatkit-sdk-specifications/blob/master/redux/state.md) (more logical order, especially if in the future we're first just fetch the current user when connecting, and joined rooms on demand)

Reply:
order applies to invocations with named params as well, like the below in `initial` fun

Reply:
👍 good point - have updated both places!